executables/collector/src/main/kotlin/de/cyface/collector/handler/MeasurementHandler.kt

```diff
@@ -0,0 +1,573 @@
+/*
+ * Copyright 2021-2022 Cyface GmbH
+ *
+ * This file is part of the Cyface Data Collector.
+ *
+ * The Cyface Data Collector is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface Data Collector is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface Data Collector. If not, see <http://www.gnu.org/licenses/>.
+ */
+package de.cyface.collector.handler
+
+import de.cyface.api.Authorizer
+import de.cyface.api.PauseAndResumeBeforeBodyParsing
+import de.cyface.api.model.User
+import de.cyface.collector.handler.exception.*
+import de.cyface.collector.model.Measurement
+import de.cyface.collector.verticle.Config
+import de.cyface.model.RequestMetaData
+import io.vertx.core.MultiMap
+import io.vertx.core.file.AsyncFile
+import io.vertx.core.file.FileProps
+import io.vertx.core.file.OpenOptions
+import io.vertx.core.http.HttpServerRequest
+import io.vertx.core.json.JsonObject
+import io.vertx.ext.auth.mongo.MongoAuthentication
+import io.vertx.ext.mongo.GridFsUploadOptions
+import io.vertx.ext.mongo.MongoClient
+import io.vertx.ext.mongo.MongoGridFsClient
+import io.vertx.ext.web.RoutingContext
+import io.vertx.ext.web.Session
+import org.apache.commons.lang3.Validate
+import org.slf4j.LoggerFactory
+import java.io.File
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.*
+import java.util.stream.Collectors
+
+/**
+ * A handler for receiving HTTP PUT requests on the "measurements" end point.
+ * This end point is the core of this application and responsible for receiving
+ * new measurements from any measurement device and forwarding those
+ * measurements for persistent storage.
```

Comment:
Does not seem like our default line break length but shorter.

Reply:
Yes. This is the default Kotlin Style.